test/files/neg/t12134/ploogin_1.scala

```diff
@@ -29,11 +29,15 @@ class Unplugged(val global: Global) extends Plugin {
     class TestPhase(prev: Phase) extends StdPhase(prev) {
       override def description = TestComponent.this.description
       def apply(unit: CompilationUnit): Unit = {
-        import scala.util.Try
         global.settings.outdir.value = s"${global.settings.plugin.value.head}/${global.settings.outdir.value}"
         val path = Paths.get(global.settings.outdir.value)
-        val perms = asFileAttribute(EnumSet.of(OWNER_READ))
-        Try(createFile(path, perms))
+        val file = path.toFile()
+        //import scala.util.Try
+        //val perms = asFileAttribute(EnumSet.of(OWNER_READ))
+        //val res = Try(createFile(path, perms))
+        //assert(res.isSuccess)
```

Comment:
Lose the comment code or justify it, IMO

Reply:
I wished to preserve my hours of researches into file creation on the JVM.